propka/lib.py

```diff
@@ -272,3 +287,31 @@ def writeFile(filename, lines):
     f.close()
 
 
+def info(*args, **kargs):
+    """Log a message. Level defaults to INFO unless overridden."""
+    level = kargs.pop("level",logging.INFO)
+    for l in _sprint(*args, **kargs):
```

Comment:
How about we move these lines into a common shared private function, and have info() and other 3 logging functions call it?